ci/scripts/clone-build_ci.sh

```diff
@@ -104,19 +94,27 @@ build_status=$?
 
 if [[ ${build_status} != 0 ]]; then
   {
-    echo "Build:                         *FAILED*"
+    echo "Build: ** FAILED ***"
     echo "Build: Failed at $(date)" || true
     echo "Build: see output at ${PWD}/log.build"
   } >> "${outfile}"
   exit "${build_status}"
 else
   {
-    echo "Build:                         *SUCCESS*"
     echo "Build: Completed at $(date)" || true
   } >> "${outfile}"
 fi
 
+export MACHINE_ID="orion"
```

Comment:
Suspect this shouldn't be hard-coded here. Isn't it already set by a detect_machine call?

Reply:
Ops, yes  as was for developing on workstation!